app/modules/components/QuickDraft.tsx

```diff
@@ -31,6 +33,7 @@ const QuickDraft = ({ buttonText, buttonStyle, refetchFn }) => {
         description: z.string(),
         type: z.string().min(1),
         license: z.string().min(1),
+        language: z.string().min(1).max(2),
```

Comment:
Apologies if you've already considered this/is wrong, but could you do:

`language: z.enum(ISO6391.getAllCodes()),`?

Reply:
Actually, just tried it and am getting some quirky TS errors, will have a look.

Reply:
I didn't know about this method - that would be much more precise validation indeed! 

Thanks for teaching me about this :)

Reply:
I've opened an issue for this to track the work (see #544). 

This PR is ready for review otherwise 😊

Reply:
OK sweet, nice one :)